jQueryI.js

```diff
@@ -0,0 +1,7 @@
+$("#Submit").click(function () {
+	alert("Your data is being processed... please wait while we file your information.");
+	const politicalAllegiance = $("#politicalAllegiance").val();
+	const citizenship = $("#citizenship").val();
+	console.log(politicalAllegiance, citizenship);
+	$("p").append(politicalAllegiance, citizenship)
```

Comment:
![bitmoji](https://render.bitstrips.com/v2/cpanel/10188118-326088164_2-s1-v1.png?transparent=1&palette=1&width=246)

Good job getting this working!!